2025-08-06 16:20:23,859 INFO    Thread-26 (_run_job):7052 [wandb_setup.py:_flush():80] Current SDK version is 0.21.0
2025-08-06 16:20:23,860 INFO    Thread-26 (_run_job):7052 [wandb_setup.py:_flush():80] Configure stats pid to 7052
2025-08-06 16:20:23,860 INFO    Thread-26 (_run_job):7052 [wandb_setup.py:_flush():80] Loading settings from C:\Users\tobys\.config\wandb\settings
2025-08-06 16:20:23,860 INFO    Thread-26 (_run_job):7052 [wandb_setup.py:_flush():80] Loading settings from c:\Users\tobys\Desktop\GBM-ML-LSTM\wandb\settings
2025-08-06 16:20:23,860 INFO    Thread-26 (_run_job):7052 [wandb_setup.py:_flush():80] Loading settings from environment variables
2025-08-06 16:20:23,860 INFO    Thread-26 (_run_job):7052 [wandb_init.py:setup_run_log_directory():703] Logging user logs to c:\Users\tobys\Desktop\GBM-ML-LSTM\wandb\run-20250806_162023-txc4a4es\logs\debug.log
2025-08-06 16:20:23,861 INFO    Thread-26 (_run_job):7052 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to c:\Users\tobys\Desktop\GBM-ML-LSTM\wandb\run-20250806_162023-txc4a4es\logs\debug-internal.log
2025-08-06 16:20:23,861 INFO    Thread-26 (_run_job):7052 [wandb_init.py:monkeypatch_ipython():622] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020FA5857B50>
2025-08-06 16:20:23,861 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():830] calling init triggers
2025-08-06 16:20:23,861 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():835] wandb.init called with sweep_config: {'batch_size': 16, 'dropout': 0.5, 'hidden_dim': 16, 'input_dim': 14, 'latent_dim': 32, 'learning_rate': 0.00022, 'num_epochs': 15, 'num_layers': 2}
config: {'_wandb': {}}
2025-08-06 16:20:23,861 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():871] starting backend
2025-08-06 16:20:24,076 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():874] sending inform_init request
2025-08-06 16:20:24,078 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():882] backend started and connected
2025-08-06 16:20:24,080 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_config_callback():1363] config_cb None None {'batch_size': 16, 'dropout': 0.5, 'hidden_dim': 16, 'input_dim': 14, 'latent_dim': 32, 'learning_rate': 0.00022, 'num_epochs': 15, 'num_layers': 2}
2025-08-06 16:20:24,080 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_label_probe_notebook():1304] probe notebook
2025-08-06 16:20:24,081 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():953] updated telemetry
2025-08-06 16:20:24,122 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():977] communicating run to backend with 90.0 second timeout
2025-08-06 16:20:28,769 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():1029] starting run threads in backend
2025-08-06 16:20:28,946 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_console_start():2458] atexit reg
2025-08-06 16:20:28,946 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-08-06 16:20:28,946 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_redirect():2375] Wrapping output streams.
2025-08-06 16:20:28,947 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_redirect():2398] Redirects installed.
2025-08-06 16:20:28,947 INFO    Thread-26 (_run_job):7052 [wandb_init.py:init():1075] run started, returning control to user process
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_finish():2224] finishing run tobiassafie-drexel-university/GBM-LSTM-dropout-latent/txc4a4es
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [jupyter.py:save_history():480] not saving jupyter history
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [jupyter.py:save_ipynb():404] not saving jupyter notebook
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [wandb_init.py:_jupyter_teardown():607] cleaning up jupyter logic
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_atexit_cleanup():2423] got exitcode: 0
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_restore():2405] restore
2025-08-06 16:25:38,997 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_restore():2411] restore done
2025-08-06 16:25:59,074 ERROR   Thread-10 (_heartbeat):7052 [internal_api.py:execute():401] [no run ID] Error executing GraphQL.
Traceback (most recent call last):
  File "c:\Users\tobys\miniconda3\envs\star-pinn\lib\site-packages\wandb\sdk\internal\internal_api.py", line 397, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "c:\Users\tobys\miniconda3\envs\star-pinn\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "c:\Users\tobys\miniconda3\envs\star-pinn\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "c:\Users\tobys\miniconda3\envs\star-pinn\lib\site-packages\wandb\sdk\lib\gql_request.py", line 59, in execute
    request.raise_for_status()
  File "c:\Users\tobys\miniconda3\envs\star-pinn\lib\site-packages\requests\models.py", line 1026, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api.wandb.ai/graphql
2025-08-06 16:26:39,188 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_footer_history_summary_info():3903] rendering history
2025-08-06 16:26:39,188 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_footer_history_summary_info():3935] rendering summary
2025-08-06 16:26:39,188 INFO    Thread-26 (_run_job):7052 [wandb_run.py:_footer_sync_info():3864] logging synced files
